designer/client/src/components/tips/Styled.tsx

```diff
@@ -2,39 +2,46 @@ import { ComponentType, SVGProps } from "react";
 import { lighten, styled } from "@mui/material";
 import { variables } from "../../stylesheets/variables";
 import { Link } from "react-router-dom";
-import { alpha } from "../../containers/theme";
+import { alpha } from "../../containers/theme/helpers";
```

Comment:
Do we need to use our custom function alpha if we can I guess the same function in mui?

Reply:
TBH I didn't want to touch these helpers in this PR because there are some Color-based helpers not available in MUI in this file and in the case of this alpha I'm not 100% sure that it's the same utility as in case of mui (I didn't check it)